pages/en/lb4/Testing-Your-Extensions.md

```diff
@@ -0,0 +1,281 @@
+---
+lang: en
+title: 'Testing your extension'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Testing-your-extension.html
+summary:
+---
+
+## Overview
+
+LoopBack 4 extensions are often used by other teams. For this reason, it is important to have a
```

Comment:
I'd say that this sentence is weak; it doesn't really tie together the idea that an extension being used by _anyone_ is a reason to test it.

IMO, this should just say something like:
```
A thorough test suite for your extension brings powerful benefits, like:
- validating extension behavior
- preventing unwanted changes to the API or functionality of your extension
- providing working samples and snippets of code that serve as functional documentation to your users
```

The exact phrasing of the bullet points doesn't have to change since the current list still captures the same ideas, but you'll want to reword them slightly to fit with the change to the previous sentence.

Reply:
I rewrote the intro slightly differently, but very close to K's text above:

LoopBack 4 extensions are often used by other teams. A thorough test suite for your extension brings powerful benefits to all your users, including:

* Validating the behavior of the extension
* Preventing unwanted changes to the API or functionality of the extension
* Providing working samples and code snippets that serve as functional documentation for your users